dashboard/app/controllers/courses_controller.rb

```diff
@@ -0,0 +1,15 @@
+class CoursesController < ApplicationController
+  def index
+    course_name = params[:course].tr('-', '_').titleize
+    course = Course.find_by_name(course_name)
```

Comment:
Nit: can `find_by_name!` here to automatically raise `ActiveRecord::RecordNotFound` instead of returning `nil` [https://apidock.com/rails/v4.0.2/ActiveRecord/FinderMethods/find_by!](https://apidock.com/rails/v4.0.2/ActiveRecord/FinderMethods/find_by!)